cli/cmd/simulation/quickexact.hpp

```diff
@@ -0,0 +1,180 @@
+//
+// Created by marcel on 06.12.23.
+//
+
+#ifndef FICTION_CMD_QUICKEXACT_HPP
+#define FICTION_CMD_QUICKEXACT_HPP
+
+#include <fiction/algorithms/simulation/sidb/minimum_energy.hpp>
+#include <fiction/algorithms/simulation/sidb/quickexact.hpp>
+#include <fiction/traits.hpp>
+#include <fiction/types.hpp>
+#include <fiction/utils/name_utils.hpp>
+
+#include <alice/alice.hpp>
+
+#include <variant>
+
+namespace alice
+{
+/**
+ *
+ */
+class quickexact_command : public command
+{
+  public:
+    /**
+     * Standard constructor. Adds descriptive information, options, and flags.
+     *
+     * @param e alice::environment that specifies stores etc.
+     */
+    explicit quickexact_command(const environment::ptr& e) :
+            command(e, "QuickExact is a quick and exact electrostatic ground state simulation algorithm designed "
+                       "specifically for SiDB layouts. It provides a significant performance advantage of more than "
+                       "three orders of magnitude over ExGS from SiQAD.")
+    {
+        add_option("--epsilon_r,-e", physical_params.epsilon_r, "Electric permittivity of the substrate (unit-less)",
+                   true);
+        add_option("--lambda_tf,-l", physical_params.lambda_tf, "Thomas-Fermi screening distance (unit: nm)", true);
+        add_option("--mu_minus,-m", physical_params.mu_minus, "Energy transition level (0/-) (unit: eV)", true);
+        add_option("--base,-b", physical_params.base,
+                   "2-state (neutral/negative) vs. 3-state (positive/neutral/negative) simulation", true);
+        add_option("--base_number_detection, -bd", params.base_number_detection,
```

Comment:
You cannot read an `enum class` value from the CLI because there is no known conversion from the stream to this particular `enum class`. You can use a `bool` and then convert it manually. However, as said above, I wouldn't even allow for the option.